EmployeeDepartment/src/main/java/com/example/EmployeeDepartment/entity/Department.java

```diff
@@ -0,0 +1,55 @@
+package com.example.EmployeeDepartment.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Entity
```

Comment:
If this is lombok annotation, it will be add the getter and setter as well. No need of additional annotations

Reply:
I have added required annotations